app/views/course/levels/_form.html.slim

```diff
@@ -0,0 +1,5 @@
+= simple_form_for [current_course, @level], resource: :course_level do |f|
+  = f.input :experience_points_threshold,
+            label: t('.experience_points_threshold'),
```

Comment:
I thought it was a little ugly that the label was not capitalised. Small thing, will remove then.